### ISSUE TITLE
ecwid module.

### ISSUE THREAD

**Issue by islatur** [hans2103/mod_ecwid_product#6]
Hi hans, just took a look at your module, 
Would it be posibble to allow all the options from the ecwid product like quantity and all that? 

thanks.

Juan

**Comment by hans2103** [comment]
Hi Juan,

Create a template override of /modules/mod_ecwid_product/tmpl/default.php to /templates/<your-template>/html/mod_ecwid_product/default.php and add <?php var_dump($data) ?> in the source. 
When you refresh the page you will see all the data you get from Ecwid.com for the given product for the given shop.

regards,
Hans

**Comment by islatur** [comment]
Hi Hans, thank sa lot for the answer, i wasn't really expecting it!  Unfortunately the overwrite didnt do the trick. 
I tried it with a free account first and it gave me an error that I don't have access to the API, then I tried with the API test shop ecwid provides and it gives me a bunch of errors.
I tried tweaking the code, but my code-fu is not the best.
here's the thread I opened for this on ecwid. http://www.ecwid.com/forums/showthread.php?t=25156

thanks a lot for your help.
